docs/integrations_terraformer.md

```diff
@@ -0,0 +1,177 @@
+# Running KICS with Terraformer for AWS
+
```

Comment:
Add an intro like so:

From version 1.5, KICS integrates with Terraformer to scan resources deployed in the cloud. The runtime information of the resources is obtained by providing a Terraformer path to KICS, via `-p` flag. The scan happens immediately after this information is obtained. In the end, results are shown as for any other KICS scan.

#### Cloud providers supported:
- AWS